src/OrbitGl/GpuTrack.cpp

```diff
@@ -114,16 +114,16 @@ float GpuTrack::GetHeight() const {
   }
   return height;
 }
+// TODO(): Make a general interface for tracks for setting the width for every child.
```

Comment:
I guess there is a bug missing

Reply:
Yes, I procrastinated that creation, thanks for pushing!